README.md

```diff
@@ -20,8 +20,16 @@ The instructions below will guide you through creating a new repository from thi
 * Set the repository visibility to public.
 * Click "Create repository from template".
 
+## Clone the repository
+
+To clone the repository with the submodule files, you can use the following command:
+
+```bash
+git clone --recursive git@github.com:openproblems-bio/<repo_name>.git
```

Comment:
fill in <repo_name> from _viash.yaml?

Reply:
this readme is not automatically generated